src/static/css/2019.css

```diff
@@ -1449,3 +1445,103 @@ p.copyright a {
     transition-delay: 0s !important;
   }
 }
+
+/* Syntax Highlighting */
+pre {
+  margin: 0;
+  border: 1px solid #ccc;
+  word-wrap: break-word;
+  padding: 6px 10px;
+  line-height: 19px;
+  margin-bottom: 20px;
+}
+
+pre code {
+  border: 0;
+  padding: 0;
+  margin: 0;
+  border-radius: 0;
+}
+
+code {
+  border: 1px solid #eaeaea;
+  margin: 0px 2px;
+  padding: 0px 5px;
+  font-size: 12px;
+}
+
+pre,
+code {
+  font-family: Consolas, 'Liberation Mono', Courier, monospace;
+  color: #333;
+  background: #f8f8f8;
```

Comment:
Let's remove this border to minimize change (and also doesn't look great on Methodology page):

```suggestion
```

Reply:
cool, I will change these things as soon as possible, Thanks @bazzadp for suggestions

Reply:
I think we should have the background to make it different from others, as I have checked removed border looks fine but when I remove the background it gets the same as earlier, and when we have background it attracts focus, it shows there is something different over there. 

@bazzadp what are your opinions?

Reply:
![image](https://user-images.githubusercontent.com/43988371/100443740-42fade00-30d0-11eb-9bb8-883ca63a33bd.png)
Without background

![image](https://user-images.githubusercontent.com/43988371/100443794-5c038f00-30d0-11eb-9f7a-30c8814d1413.png)
with background

Reply:
I pondered this myself. In some ways it helps differentiate it. In other ways, because the code snippets are small it's not really needed.

However the reason I ultimately suggested removing it are because of the knock on effects to other places:
- The Methodology SQL code snippet. Look at that before and after. Definitely looks worse with background because we have the floating card. Now maybe we should replace the floating card with this instead but I was trying to minimise the changes in this as launch date is fast approaching. And the next point:
- In the Table of Contents for chapters with code in headers. For example the 2019 Caching chapter.

Do you see what I mean?

Reply:
Methdology SQL before and after:

![image](https://user-images.githubusercontent.com/10931297/100450610-e5559a00-30ad-11eb-8077-f66df1788134.png)

![image](https://user-images.githubusercontent.com/10931297/100450631-ed153e80-30ad-11eb-9cbe-43855f0180c1.png)

Table of Contents on Caching chapter before and after:

![image](https://user-images.githubusercontent.com/10931297/100450692-06b68600-30ae-11eb-95b7-8b0550ed3163.png)

![image](https://user-images.githubusercontent.com/10931297/100450892-5dbc5b00-30ae-11eb-91f4-e187542228d3.png)

Reply:
okay cool, I will remove the background, will push newly updated